build.gradle

```diff
@@ -4,24 +4,29 @@ plugins {
     id 'maven'
 }
 
-sourceCompatibility = '1.7'
+group = 'uk.sky.cirrus'
+sourceCompatibility = '1.8'
+
+repositories {
+    mavenCentral()
+}
 
 task wrapper(type: Wrapper) {
     gradleVersion = '2.7'
 }
 
 dependencies {
-    compile 'com.datastax.cassandra:cassandra-driver-core:2.0.3'
+    compile 'com.datastax.cassandra:cassandra-driver-core:2.1.8'
     compile 'com.google.guava:guava:18.0'
```

Comment:
Same with guava. This is probably okay as long as we don't depend on @Beta apis.

It's a relatively common pattern (e.g. mockito, guice) to bundle guava internally.